Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM  --platform=linux/amd64 python:3.8-slim-bullseye
```

Comment:
@zimzoom mentioned we actually updated to python3.9 but forgot to update the README. Can you update this?

Reply:
👍